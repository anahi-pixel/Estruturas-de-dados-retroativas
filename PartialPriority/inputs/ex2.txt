1 1 100
1 2 200
7
8
5
6
3 1
7 
8
5
6
