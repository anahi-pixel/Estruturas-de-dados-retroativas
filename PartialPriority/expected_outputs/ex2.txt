1 Fila de prioridade no instante atual: 
100 200 
   1:0 0 100
1:0 0 2147483647
   2:0 0 200

100
2
2 Fila de prioridade no instante atual: 
200 
2:0 0 200

200
1
